lib/backend/k8s/conversion/conversion.go

```diff
@@ -135,7 +126,7 @@ func (c Converter) isScheduled(pod *kapiv1.Pod) bool {
 	return pod.Spec.NodeName != ""
 }
 
-func (c Converter) isHostNetworked(pod *kapiv1.Pod) bool {
+func (c Converter) IsHostNetworked(pod *kapiv1.Pod) bool {
```

Comment:
Why did you make this one public but not the others? e.g. `isScheduled`, etc?

Reply:
I just updated what I needed to get things working.

Reply:
but i can make em public.